logstash/tests/logstash_test.py

```diff
@@ -579,14 +579,6 @@ def test_setting_fullnameOverride():
     r = helm_template(config)
 
     custom_name = 'logstash-custom'
-    assert custom_name in r['daemonset']
-    assert r['daemonset'][custom_name]['spec']['template']['spec']['containers'][0]['name'] == project
-    assert r['daemonset'][custom_name]['spec']['template']['spec']['serviceAccountName'] == name
-    volumes = r['daemonset'][custom_name]['spec']['template']['spec']['volumes']
-    assert {
-               'name': 'data',
-               'hostPath': {
-                   'path': '/var/lib/' + custom_name + '-default-data',
-                   'type': 'DirectoryOrCreate'
-               }
-           } in volumes
+    assert custom_name in r['statefulset']
+    assert r['statefulset'][custom_name]['spec']['template']['spec']['containers'][0]['name'] == 'logstash'
+    volumes = r['statefulset'][custom_name]['spec']['template']['spec']['volumes']
```

Comment:
The last line `volumes = ...` doesn't seem to be used

Reply:
Where I copied it from it was used for this check
```
    assert {
               'name': 'data',
               'hostPath': {
                   'path': '/var/lib/' + custom_name + '-default-data',
                   'type': 'DirectoryOrCreate'
               }
           } in volumes
```
I don't think we care about custom_name being in volume mount path... So I'm going to delete this.

Reply:
I rebased  my branch on upstream/master, amended this change into the _Update tests_ commit and force-pushed